src/azstoragetorch/_client.py

```diff
@@ -58,6 +58,8 @@ def __init__(
     ):
         self._sdk_blob_client = sdk_blob_client
         self._generated_sdk_storage_client = self._sdk_blob_client._client
+        self._sdk_blob_client._config.user_agent_policy.add_user_agent("azstoragetorch/0.0.1")
```

Comment:
For the version value, we should try to import if from [`_version.py`](https://github.com/Azure/azure-storage-for-pytorch/blob/490658e58084dd06e8c078d9ac3cadd45810d7f9/src/azstoragetorch/_version.py#L7) so the user agent is automatically updated with bump of the library's version.